src/custom-elements/src/index.html

```diff
@@ -3,15 +3,268 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0">
-  <link rel="stylesheet" href="http://localhost:8000/chi.css">
-  <title>Stencil Component Starter</title>
-  <script src="/build/ux-chi-ce.js"></script>
 
-</head>
-<body>
+  <!-- CHOOSE WHERE TO GET CHI STYLESHEET -->
+
+  <!--  LOCALHOST: if you have to test style changes -->
+  <!-- <link rel="stylesheet" href="http://localhost:8000/chi.css"> -->
+
+  <!--  PRODUCTION: no need to test style changes -->
+  <link rel="stylesheet" href="https://assets.ctl.io/chi/5.43.0/chi.css">
+  <script src="https://assets.ctl.io/chi/5.43.0/js/chi.js" integrity="sha256-QrqSLxUsQwxddEI9YvRqsMWwEqkADI9P5z9mBHfqaFQ=" crossorigin="anonymous"></script>
+
+  <link rel="icon" href="assets/favicon.ico">
+
+
+  <title>Chi - Custom elements</title>
+  <script type="module" src="/build/ux-chi-ce.esm.js"></script>
+  <script nomodule src="/build/ux-chi-ce.js"></script>
 
-  <em>Place you element here</em>
-  <chi-steps></chi-steps>
+</head>
+<body class="-p--5">
 
 </body>
 </html>
+
+<script>
+  // ADDS ALL COMPONENTS TO BODY
+  const components = { 
```

Comment:
I don't know if we want to test components locally that way instead of using tags directly inside the `<body>`, what do you think @jllr?